locale/en/LC_MESSAGES/django.po

```diff
@@ -290,53 +629,110 @@ msgstr "Maintenance"
 msgstr "Input an URL identifier for this module."
 
+#: course/models.py exercise/exercise_models.py
+msgid "LABEL_POINTS_TO_PASS"
+msgstr "points to pass"
+
 #: course/models.py
-msgid "MODULE_READING_OPENING_TIME_VERBOSE"
-msgstr "Opening time for the reading material"
+msgid "LABEL_INTRODUCTION"
+msgstr "introduction"
+
+#: course/models.py
+msgid "LABEL_READING_OPENING_TIME"
+msgstr "opening time for the reading material"
 
 #: course/models.py
 msgid "MODULE_READING_OPENING_TIME_HELPTEXT"
 msgstr ""
-"Leave empty if the reading material should not open before the exercises."
+"Leave empty if the reading material should not open before the assignments."
+
+#: course/models.py
+msgid "LABEL_EXERCISE_OPENING_TIME"
+msgstr "opening time for the assignments"
+
+#: course/models.py
+msgid "LABEL_CLOSING_TIME"
+msgstr "closing time"
+
+#: course/models.py
+msgid "MODULE_CLOSING_TIME_HELPTEXT"
+msgstr "The default closing time of the assignments in this module."
+
+#: course/models.py
+msgid "LABEL_LATE_SUBMISSIONS_ALLOWED"
+msgstr "late submissions allowed"
+
+#: course/models.py
+msgid "LABEL_LATE_SUBMISSION_DEADLINE"
+msgstr "late submission deadline"
+
+#: course/models.py
+msgid "LABEL_LATE_SUBMISSION_PENALTY"
+msgstr "late submission penalty"
 
 #: course/models.py
 msgid "MODULE_LATE_SUBMISSION_PENALTY_HELPTEXT"
-msgstr "Multiplier of points to reduce, as decimal. 0.1 = 10%"
+msgstr "Multiplier of points to reduce as a decimal (e.g. 0.1 = 10%)."
+
+#: course/models.py
+msgid "MODEL_NAME_COURSE_MODULE"
+msgstr "course module"
+
+#: course/models.py
+msgid "MODEL_NAME_COURSE_MODULE_PLURAL"
+msgstr "course modules"
 
 #: course/models.py
 msgid "MODULE_ERROR_OPENING_TIME_AFTER_CLOSING_TIME"
-msgstr "Opening time must be earlier than the closing time."
+msgstr "The opening time must be earlier than the closing time."
 
 #: course/models.py
 msgid "MODULE_ERROR_LATE_SUBMISSION_DL_BEFORE_CLOSING_TIME"
-msgstr "Late submission deadline must be later than the closing time."
+msgstr "The late submission deadline must be later than the closing time."
 
 #: course/models.py
 msgid "MODULE_ERROR_READING_OPENING_TIME_AFTER_EXERCISE_OPENING"
 msgstr ""
-"Opening time of reading material must be earlier than the opening time of "
-"the exercises."
+"The opening time of reading material must be earlier than the opening time "
+"of the assignments."
 
 #: course/models.py
 msgid "STATUS_NO_TOTAL_POINTS"
 msgstr "No total points"
 
+#: course/models.py
+msgid "LABEL_CONFIRM_THE_LEVEL"
+msgstr "confirm the level"
+
 #: course/models.py
 msgid "LEARNING_OBJECT_CATEGORY_LEVEL_CONFIRMATION_EXERCISE_HELPTEXT"
 msgstr ""
-"Once exercise is graded non zero it confirms all the points on the hierarchy "
-"level. Implemented as a mandatory feedback feature."
+"Once an assignment is graded non-zero, it confirms all the points on the "
+"hierarchy level (e.g. within the chapter or the module). Implemented as a "
+"mandatory feedback feature."
```

Comment:
Is this string visible to end-users? The info "Implemented as a mandatory feedback feature." is quite technical in that context

Reply:
It is, it's visible to teachers when defining category settings (as a help text for the "Confirm the level" (the name isn't very easy to understand):
![image](https://user-images.githubusercontent.com/50318434/130036473-a3b80c72-f630-4232-ad46-17f457c8deb5.png)
I think the phrase is quite techincal and awkward (and difficult to understand), do you have any better ideas how to get the point across? (I think even the "on the hierarchy level" is difficult to understand, which is why hopefully the added specification helps.)

Reply:
I’d label the feature according to what it’s meant for, i.e., making things mandatory.  Maybe something like:
_[ ] **Mandatory component**
Students must have non-zero points in this category before their other points in the same chapter/module become official. Suggested usage: check this box for feedback activities that students must fill to score points for a course component._

Reply:
Sounds great, thanks!

Reply:
Incidentally, there’s an unrelated minor UX issue in that screenshot: the pane is titled just "Category" and there is no indication of which category is being edited.

Reply:
Another off-topic comment now that I’m at this: I’ve just realized that even after using A+ for years, I don’t know what each of the options in the Status menu does. Maybe don’t need to know, and I guess there’s documentation for that somewhere, but this view could also be more helpful.

Reply:
>Incidentally, there’s an unrelated minor UX issue in that screenshot: the pane is titled just "Category" and there is no indication of which category is being edited.

Good point! I suppose the "Name" field is how to distinguish between the categories, but since it's not first in the list, it doesn't seem like an identifier.

Reply:
>(EDIT: The existing text says "non-zero", as does my suggestion, but is that actually correct? Does the "Points to pass" setting affect this?)

Excellent point! I'm not entirely sure. Does @Mankro know?

Reply:
I haven't tried that. Since the old text says "non-zero", let's keep that.